packages/api/portfolioDrafts/funding/createFundingStep.ts

```diff
@@ -73,7 +76,7 @@ export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayPr
         Key: {
           id: portfolioDraftId,
         },
-        UpdateExpression: "set #stepKey = :step, #updateAtKey = :now",
+        UpdateExpression: "set #stepKey = :step, #updateAtKey = :now, num_task_orders = :numOfTaskOrders",
```

Comment:
This works for now during the Create but we've got to be careful about this (and similar) things during the the Updates or if we add additional task orders. I don't really remember if that's implemented yet. But we're going to need to look into [atomic counters](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithItems.html#WorkingWithItems.AtomicCounters) once we get closer to that.